### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by richgilbank**
Correcting file paths

### CODE REVIEW

File: data/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var self = require('sdk/self');
 
-// a dummy function, to show how tests work. 
-// to see how to test this function, look at ../test/test-main.js
+// a dummy function, to show how tests work.
+// to see how to test this function, look at test/test-index.js
```
Comment (erikvold) [inline data/index.js]: `test/test-index.js` -> `../test/test-index.js` please

Comment (richgilbank) [inline data/index.js]: Running `jpm init` seems to create the following structure:

Is this  used outside of init as well?
https://github.com/mozilla/jpm/blob/master/lib/init.js#L56-L57

Comment (erikvold) [inline data/index.js]: yes the tests are used for `jpm test`, for instance try `jpm init && jpm test`

Comment (richgilbank) [inline data/index.js]: Right, it's used in the tests, but that doesn't modify the directory structure at any point, so `index.js` is always a sibling of the `test` directory and getting to `test-index.js` would just be `./test/test-index.js`, no? I think I'm missing something here. Happy to change it if you insist, but I don't see any scenario where `test` is up a directory.

Comment (erikvold) [inline data/index.js]: ah you're right, thanks!

**Pull request merged**